lang/LangPrimSource/PyrPrimitive.cpp

```diff
@@ -3645,7 +3645,10 @@ static int prLanguageConfig_writeConfigFile(struct VMGlobals* g, int numArgsPush
         config_path = SC_Codecvt::utf8_str_to_path(path);
         gLanguageConfig->writeLibraryConfigYAML(config_path);
```

Comment:
whoops, just realized this results in a superfluous write -- can you remove this line?